terraform/ecr-sandbox/root.tf

```diff
@@ -0,0 +1,7 @@
+resource "aws_ecr_repository" "sandbox_ecr_repository" {
```

Comment:
Has terraform fmt been run on this file?

Reply:
I should have spotted the build failure, thanks!